### ISSUE TITLE
Use MongoDB TTLs

### ISSUE THREAD

**Issue by JacobEvelyn** [codeforboston/mbta-ninja#45]
I haven't looked at your code at all but my Meteor output says you're using a cron job to expire old events. My guess is this is really what you want: http://docs.mongodb.org/manual/tutorial/expire-data/

I've used it before and it works like a charm. :)

**Comment by JacobEvelyn** [comment]
Ah, maybe this isn't what you want if you're treating expired things as different from nonexistent things.

**Comment by davidlago** [comment]
Thanks @JacobEvelyn. That's the first thing I tried, but it doesn't work well because although the tasks are expired, there is no guarantee when they will be "garbage collected". In a default MongoDB installation, it is fairly quick (it removes expired documents every minute), but it turns out that there is no controlling when it will happen in our hosted mongo at MongoLab (and the tests we ran yesterday confirmed that they didn't expire for at least 1 hour).

Also, we might want to keep the events around in case we want to do some post-analysis of the data.
